<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="4" column="3">
    <widget class="QGroupBox" name="phaseSelectionWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Phases (if available)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QSpinBox" name="parallelProcessesSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QPushButton" name="valveTypeSelectionButton">
     <property name="text">
      <string>Select/Deselect All</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="ctkCollapsibleGroupBox" name="exportOptionsFrame">
     <property name="title">
      <string>Export Plugins</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2"/>
    </widget>
   </item>
   <item row="13" column="0" colspan="4">
    <widget class="QProgressBar" name="progressbar">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="format">
      <string>%v / %m</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="ctkPathLineEdit" name="outputDirSelector">
     <property name="toolTip">
      <string>Directory that will contain exported data.</string>
     </property>
     <property name="filters">
      <set>ctkPathLineEdit::Dirs|ctkPathLineEdit::NoDot|ctkPathLineEdit::NoDotDot|ctkPathLineEdit::Readable</set>
     </property>
     <property name="settingKey">
      <string>ValveBatchExportOutputDir</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="phaseSelectionButton">
     <property name="text">
      <string>Select/Deselect All</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Input directory:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="4">
    <widget class="QPlainTextEdit" name="statusLabel"/>
   </item>
   <item row="9" column="3">
    <widget class="QPushButton" name="maxProcessesButton">
     <property name="toolTip">
      <string>Maximum number of processes</string>
     </property>
     <property name="text">
      <string>max</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QGroupBox" name="createValvesGroupBox">
     <property name="title">
      <string>Create Valves</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="intermediateRadioButton">
        <property name="toolTip">
         <string>If checked, sequence frames that don't have a heart valve assigned, will receive a new heart valve. 

Example: frame 4 and frame 17 have a heart valve assigned.  For each intermediate frame (5 to 16) a new heart valve will be created and added to the export. </string>
        </property>
        <property name="text">
         <string>Intermediate</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="wholeSequenceRadioButton">
        <property name="text">
         <string>Whole Sequence</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Output directory:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="4">
    <widget class="QPushButton" name="exportButton">
     <property name="toolTip">
      <string>Iterate through all the scene files in the input directory and write results to the output directory</string>
     </property>
     <property name="text">
      <string>Start Export</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="ctkPathLineEdit" name="inputDirSelector">
     <property name="toolTip">
      <string>Directory containing scene (.mrb) files with heart valves.</string>
     </property>
     <property name="filters">
      <set>ctkPathLineEdit::Dirs|ctkPathLineEdit::NoDot|ctkPathLineEdit::NoDotDot|ctkPathLineEdit::Readable</set>
     </property>
     <property name="settingKey">
      <string>ValveBatchExportInputDir</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Parallel processes:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QPushButton" name="minProcessesButton">
     <property name="toolTip">
      <string>Minimum number of processes</string>
     </property>
     <property name="text">
      <string>min</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QGroupBox" name="valveTypeSelectionWidget">
     <property name="title">
      <string>Valve Types (if available)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
